prow/api-resources.py

```diff
@@ -109,18 +109,18 @@ def main(delimiter, format, group_blacklist, kind_blacklist):
         "--format", type=str, default="%s", help="format string for each api resource."
     )
     parser.add_argument(
-        "--group-blacklist",
+        "--group-denylist",
```

Comment:
Aren't these breaking changes? I would have expected both flags to be supported temporarily or for the uses of the script to be updated in this PR as well.

Reply:
I couldn't find these flags being used in any Istio repo; it seems like only the default value is ever used

Reply:
See for example #2551

Reply:
Yes, I added these flags to make this configurable but made is so we are using the default. So this should not be breaking for Istio.